lib/elixir/lib/enum.ex

```diff
@@ -461,6 +461,26 @@ defmodule Enum do
     end) |> elem(1)
   end
 
+
+  @doc """
+  Enumerates the collection, removing adjacent duplicated elements.
```

Comment:
WDYT about "consecutive duplicated elements" instead of "runs of duplicated elements"? I think using the "run of" expression like this is quite uncommon.